include/swift/Runtime/Mutex.h

```diff
@@ -10,124 +10,137 @@
 //
 //===----------------------------------------------------------------------===//
 //
-// Mutex, Condition, and Scoped lock abstactions for use in Swift runtime.
+// Mutex, Condition, Read/Write lock, and Scoped lock abstactions
+// for use in Swift runtime.
 //
 //===----------------------------------------------------------------------===//
 
 #ifndef SWIFT_RUNTIME_MUTEX_H
 #define SWIFT_RUNTIME_MUTEX_H
 
-#if (defined(__APPLE__) || defined(__linux__) || defined(__CYGWIN__) || defined(__FreeBSD__))
-#define SWIFT_RUNTIME_MUTEX_HAVE_PHTREADS
+#if (defined(__APPLE__) || defined(__linux__) || defined(__CYGWIN__))
```

Comment:
Yeah I just noticed that... need to get my preferred merge tool installed on my laptop (away from home again).